docs/migration-guides/1.22-1.23.md

```diff
@@ -0,0 +1,46 @@
+# Migrate from 1.22 to 1.23
+
+## Increase the Loki volume size
+In Kyma 1.23.0, the Loki volume size requested by the persistent volume claim has been increased from 10Gi to 30Gi
+
+## Upgrade process
+Before starting the upgrade steps mentioned in one of the scenario needs to be followed.
+### Custom Loki volume size
+
+If the Loki volume size requested by the persistence volume claim is different from out of box setting (which is 10Gi): If the size is > 30 Gi please use kyma upgarade command by passing the overrides for the desired persistent volume claim size as shown [here](https://kyma-project.io/docs/cli/#details-kyma-cli-command-usage-examples-upgrade-kyma)
+
+Below is an example for override file:
+```yaml
+apiVersion: v1
+data:
+  loki.persistence.size: "30Gi"
+kind: ConfigMap
+metadata:
+  labels:
+    component: logging
+    installer: overrides
+  name: logging-overrides
+  namespace: kyma-installer
+```
+
+If the size is between 10Gi and 30Gi please follow the steps mentioned in scenario below.
+
+### Out of box Loki volume size
+
+However if the persistence volume claim is out of box setting use following command to expand the volume:
+
+```
+kubectl delete statefulsets.apps -n kyma-system logging-loki
+
+kubectl patch persistentvolumeclaims -n kyma-system storage-logging-loki-0 --patch '{"spec": {"resources": {"requests": {"storage": "30Gi"}}}}'
+```
+>Volume expansions are only possible if the field `allowVolumeExpansion` of the corresponding storage class is set to true on the storage class. Please look [here]
+
+If the above process fails then the persistent volume claim template needs to be deleted. Use following command:
+
+```
+kubectl delete statefulsets.apps -n kyma-system logging-loki
+
+kubectl delete persistentvolumeclaims -n kyma-system storage-logging-loki-0
+```
+> This action will delete the existing data.
```

Comment:
Can we add the information how the users see that the overall operation was successful? Either here, at the end of the entire document, or after the respective instructions that work if there's no error.

And what is the next step? Line 7 announces that the instructions are a preparation for the upgrade, so can we say that the users can now proceed to do the upgrade to 1.23.0?